app/views/admin/jobs/index.html.erb

```diff
@@ -21,6 +21,24 @@
   </tbody>
 </table>
 
+<% if @error_calendars.count > 0 -%>
+<h2><%= @error_calendars.count %> Error Calendars</h2>
+<% @error_calendars.order(:name).each do |cal| -%>
+  <h3><%= link_to cal.name, admin_calendar_path(cal) -%></h3>
```

Comment:
currently links to `http://admin.lvh.me:3000/calendars/76` should it be `http://admin.lvh.me:3000/calendars/76/edit`